api/src/main/java/org/openmrs/module/fhir/api/strategies/medication/MedicationStrategy.java

```diff
@@ -10,11 +10,14 @@
 import org.openmrs.api.context.Context;
 import org.openmrs.module.fhir.api.util.FHIRMedicationUtil;
 import org.openmrs.module.fhir.api.util.FHIRUtils;
+import org.openmrs.module.fhir.api.util.StrategyUtil;
 import org.springframework.stereotype.Component;
 
 import java.util.ArrayList;
 import java.util.List;
 
+import static org.openmrs.module.fhir.api.util.FHIRUtils.extractUuid;
```

Comment:
Is it used here?

Reply:
No, I can remove it, but as I mentioned to @areklalo I'm going to automatically optimize import in my next PR

Reply:
Sure. I think you can do it later then :)